browser/src/control/Control.DownloadProgress.js

```diff
@@ -114,12 +111,14 @@ L.Control.DownloadProgress = L.Control.extend({
 		var buttonText = _('Copy') + L.Util.replaceCtrlAltInMac(' (Ctrl + C)');
 
 		if (inSnackbar) {
-			this._map.uiManager.showProgressBar(snackbarMsg, buttonText,
-				this._onConfirmCopyAction.bind(this), this.options.snackbarTimeout);
-
 			this._map.uiManager.setSnackbarProgress(100);
 
-			this.setupKeyboardShortcutForSnackbar();
+			this._map.uiManager.showSnackbar(
```

Comment:
You also replace the progressbar with a snackbar here, which is not mentioned in the commit message. But it's probably OK. Could you please describe why do you change it, though?

Reply:
Added a note.